### ISSUE TITLE
annual reports summary & individual pages - elements with no accessible name/alt text

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#108]
https://rrchnm.org/our-story/annual-reports/

The 3 image links on this page are missing accessible names. Their individual summary pages also have the same alert.

This issue also comes up for the avatar on the summary pages.

**Comment by kimisgold** [comment]
The file previews should now have alt text describing them as the cover pages for their respective documents.

**Comment by IdaAkiwumi** [comment]
The names are similar as they are all showing 2017-2018

![Screen Shot 2020-10-14 at 4 36 10 PM](https://user-images.githubusercontent.com/618307/96042604-a63ff200-0e3b-11eb-9a90-7c50cc16abeb.png)

**Comment by kimisgold** [comment]
Whoops, should be fixed now.
